server/modules/syllabus/commands/find-syllabus-by-code/find-syllabus-by-code-controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Request, Response, NextFunction } from 'express'
+
+import { UnauthorizedException } from '../../../../libs/exceptions/unauthorized-exception'
+import { Syllabus } from '../../domain/syllabus'
+import { FindSyllabusByCodeHttpRequest } from './find-syllabus-by-code-dto'
+import { FindSyllabusByIdService } from './find-syllabus-by-code-service'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const FindSyllabusByCodeController = async (req: Request, res: Response, next: NextFunction) => {
+  if (!req.user || !req.user.acceptedPolicies) {
+    const error = new UnauthorizedException()
+    res.status(error.code)
+    return res.json(error)
+  }
```

Comment:
Currently, there is no need to restrict this functionality to authenticated users only.